digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectSectionCalculator.scala

```diff
@@ -232,6 +232,7 @@ object ProjectSectionCalculator {
         case (_, Some(LinkStatus.UnChanged), _) => nextSection.map(_.startAddrM)
         case (LinkStatus.NotHandled, _, Some(value)) => Some(value + currentSection.endAddrM - currentSection.startAddrM)
         case (LinkStatus.Transfer, _, Some(value)) => Some(value + currentSection.endAddrM - currentSection.startAddrM)
+        case (LinkStatus.Numbering, _, Some(value)) => Some(value + currentSection.endAddrM - currentSection.startAddrM)
```

Comment:
Numbering should just have currentSection.endAddrM as it shouldn't affect the values at all.

Reply:
I'll change this